frontend/src/components/institute/NewSchoolPage.vue

```diff
@@ -683,10 +683,10 @@ export default {
       return this.schoolReportingRequirementTypeCodes ? this.schoolReportingRequirementTypeCodes : [];
     },
     canOnlyAddIndependentSchools() {
-      return this.hasRequiredPermission(this.userInfo, PERMISSION.EDIT_INDEPENDENT_SCHOOL_PERMISSION) 
+      return this.hasRequiredPermission(this.userInfo, PERMISSION.EDIT_INDEPENDENT_SCHOOL_PERMISSION) && !this.hasRequiredPermission(this.userInfo, PERMISSION.EDIT_SCHOOL_PERMISSION);
```

Comment:
& !this.hasRequiredPermission(this.userInfo, PERMISSION.EDIT_OFFSHORE_SCHOOL_PERMISSION)